zbus/tests/basic.rs

```diff
@@ -117,6 +117,24 @@ async fn fdpass_systemd_async() {
     f.metadata().unwrap();
 }
 
+#[cfg(all(target_os = "linux", feature = "ibus"))]
+#[test]
+fn ibus() {
+    use zbus::{blocking, conn};
+
+    let ibus_blocking = blocking::connection::Builder::ibus()
+        .unwrap()
+        .build()
+        .unwrap();
+    let ibus_non_blocking = block_on(block_on(conn::Builder::ibus()).unwrap().build()).unwrap();
```

Comment:
the blocking wrapper does exactly this, so I don't think this is needed.

Reply:
The blocking api will call the non-blocking one. It is redundancy to call it 1 more time.

Reply:
I'm not sure I follow. You wrote the same thing I did, in different words? :thinking: